src/style/theme.ts

```diff
@@ -27,7 +27,7 @@ const defaultThemeOptions: ThemeOptions = {
     h1: {
       fontSize: 20,
       fontColor: darkGrey,
-      fontWeight: 400,
+      fontWeight: 900,
```

Comment:
This will effect all text in the website that uses Body1. You shouldn't change this file.
You can see an example for how to correctly change the font weight in src/components/molecules/TextView/TextViewRecord.tsx